photon-colors/build.js

```diff
@@ -27,31 +27,30 @@ const formats = {
     'ext': 'css'
   },
   'less': {
-    'output': [`/* Firefox Colors Less Variables v${metadata.version} */
+    'output': [`/* Photon Colors Less Variables v${metadata.version} */
 
 `],
     'formatter': (color, variant, value) => `@${color}-${variant}: ${value};\n`,
     'ext': 'less'
   },
   'sass': {
-    'output': [`/* Firefox Colors SCSS Variables v${metadata.version} */
+    'output': [`/* Photon Colors SCSS Variables v${metadata.version} */
 
 `],
     'formatter': (color, variant, value) => `$${color}-${variant}: ${value};\n`,
     'ext': 'scss'
   },
   'js': {
-    'output': [`/* Firefox Colors JS Variables v${metadata.version} */
+    'output': [`/* Photon Colors JS Variables v${metadata.version} */
 
 `],
     'formatter': (color, variant, value) => `exports.${color.toUpperCase()}_${variant} = '${value}';\n`,
     'ext': 'js'
   },
   'gimp': {
     'output': [`GIMP Palette
-Name: Firefox/Photon
-# Firefox Colors GPL Color Palette v${metadata.version}
-# ${metadata.homepage}
+Name: Photon Colors
+# Photon Colors GPL Color Palette v${metadata.version}
 
```

Comment:
Could we add the homepage back?

Reply:
Sure thing! I removed it because during my test it was never visible and it was the only file that featured it.